src/client/components/email-form/email-form.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <form
+    v-if="emailAddress"
+    :action="`https://formspree.io/${ emailAddress }`"
+    class="email-form"
+    method="POST">
+    {{ nextUrl }}
+    <input type="hidden" name="_next" :value="nextUrl" />
+    <input type="hidden" name="_subject" :value="form.title" />
+    <input type="hidden" name="_language" :value="$i18n.locale" />
+    <input type="text" name="_gotcha" style="display:none" />
+
+    <h2 class="email-form__title">{{ form.title }}</h2>
+    <rich-text :id="messageId" :text="form.confirmationMessage" class="alert alert--success email-form__alert" />
```

Comment:
Should we make an "alert box" component?:
```vue
<alert-box type="success">
  <rich-text :id="messageId" :text="form.confirmationMessage" />
</alert-box>
```
And also move the styling there.

Reply:
I wanted to do that in another ticket, otherwise it takes longer to get this ticket done. Also for the burndown chart ;)

Reply:
@jbmoelker scope creeping ;)

Reply:
✅